### PR TITLE
TestLoader's support for custom paths

### PR DESCRIPTION
**Pull request by GeisaFaustino**
Add support for custom paths (seismic and labels) for both, training and testing purposes.

### CODE REVIEW

**Comment by MJZawacki** [comment]
@maxkazmsft We are getting CUDA out of memory errors on this PR build. Is there an issue with the build machine?

File: interpretation/deepseismic_interpretation/dutchf3/data.py [hunk 1]
```diff
@@ -246,7 +246,17 @@ def get_random_batch(
...
-    def __init__(self, data_dir, split="train", is_transform=True, augmentations=None):
+    """
+    Base class for section data loader
+    :param str data_dir: Root directory for training/test data
+    :param str split: split file to use for loading patches
+    :param bool is_transform: Transform patch to dimensions expected by PyTorch
+    :param list augmentations: Data augmentations to apply to patches
+    :param str seismic_path: Override file path for seismic data
+    :param str label_path: Override file path for label data
+    """
```
Comment (yalaudah) [inline interpretation/deepseismic_interpretation/dutchf3/data.py]: Sweet! I love this :)

File: interpretation/deepseismic_interpretation/dutchf3/data.py [hunk 2]
```diff
@@ -576,7 +656,18 @@ def transform(self, img, lbl):
...
-    def __init__(self, data_dir, stride=30, patch_size=99, is_transform=True, augmentations=None):
+    """
+    Test Data loader for the patch-based deconvnet
+    :param str data_dir: Root directory for training/test data
+    :param int stride: training data stride
+    :param int patch_size: Size of patch for training
+    :param bool is_transform: Transform patch to dimensions expected by PyTorch
+    :param list augmentations: Data augmentations to apply to patches
+    :param str seismic_path: Override file path for seismic data
+    :param str label_path: Override file path for label data
```
Comment (yalaudah) [inline interpretation/deepseismic_interpretation/dutchf3/data.py]: these parameters don't exist in the __init__(). Can they be removed?

**Comment by maxkazmsft** [comment]
> 
> 
> @maxkazmsft We are getting CUDA out of memory errors on this PR build. Is there an issue with the build machine?

Not that I know of... could you please point me to a build log?

File: interpretation/deepseismic_interpretation/dutchf3/data.py (hunk 1 above)
Comment (maxkazmsft) [reply interpretation/deepseismic_interpretation/dutchf3/data.py]: +1 finally someone added comments

File: interpretation/deepseismic_interpretation/dutchf3/data.py (hunk 2 above)
Comment (GeisaFaustino) [reply interpretation/deepseismic_interpretation/dutchf3/data.py]: @yalaudah I can do this. :-) However, we have to close this PR and reopen a new one pointing out to the new contrib branch.

**Comment by MJZawacki** [comment]
@maxkazmsft It looks like the most recent commit build passed the build so I'm assuming it was an intermittent issue. If you care to follow up, here's the original failure I was referring to: https://dev.azure.com/best-practices/deepseismic/_build/results?buildId=22692&view=logs&j=3b073ba7-9163-54af-bcff-dd67864dc387&t=f90b85f0-6e14-5123-336c-e383a1694d19&l=185

**Pull request closed without merging**